### ISSUE TITLE
Look into Govert's Dockerfile rework

### ISSUE THREAD

**Issue by greenw0lf** [beeldengeluid/whisper-asr-worker#84]
See if I can adapt it to the new code and if it works + has benefits compared to the current approach, otherwise drop it for now.

**Comment by greenw0lf** [comment]
Tried to adapt and it worked on the CPU, but as expected I had trouble running it on the GPU. I tried to install the CUDA dependencies needed (because they are definitely not present) via
```
pip install nvidia-cublas-cu12 nvidia-cudnn-cu12==8.9.7.29
export LD_LIBRARY_PATH=`python3 -c 'import os; import nvidia.cublas.lib; import nvidia.cudnn.lib; print(os.path.dirname(nvidia.cublas.lib.__file__) + ":" + os.path.dirname(nvidia.cudnn.lib.__file__))'`
```
(from https://github.com/SYSTRAN/faster-whisper/)

and that failed, so I also tried just installing `torch`. Neither approach worked (plus, the image size is larger than the currently active one) so I will leave this aside for now and focus on more important issues.
